source/install/install-tar.rst

```diff
@@ -107,6 +107,17 @@ Why does Mattermost fail to start at system boot?
 
 If your database is on the same system as your Mattermost Server, we recommend editing the default ``/lib/systemd/system/mattermost.service`` systemd unit file to add ``After=postgresql.service`` and ``BindsTo=postgresql.service`` to the ``[Unit]`` section.
 
-.. tip::
-	
-	We recommend the `Mattermost Omnibus install method </install/installing-mattermost-omnibus.html>`__ over the tarball if you are running the Mattermost Server and database a single system as this greatly reduces setup and ongoing maintenance.
+Can I run Mattermost without a proxy?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Yes. However, like NGINX, in place, Mattermost binds to 443 instead of 8065. The Mattermost binary requires the correct permissions to do that binding. You must activate the ``CAP_NET_BIND_SERVICE`` capability to allow the new Mattermost binary to bind to ports lower than 1024 by running the following command:
+
+.. code-block:: none
+  :class: mm-code-block 
+
+    sudo setcap cap_net_bind_service=+ep ./mattermost/bin/mattermost
+
+.. note::
+  
+  - We don't recommend deploying Mattermost on a single server unless you have fewer than 200 users.
+  - We recommend the `Mattermost Omnibus install method </install/installing-mattermost-omnibus.html>`__ over the tarball if you are running the Mattermost Server and database a single system as this greatly reduces setup and ongoing maintenance.
```

Comment:
We should add a `**Ubuntu Only` to this because it's generic tar docs. I wouldn't want to mislead here.

Side note, the omnibus page has no info on what OSs it supports

Reply:
@nab-77 - The Omnibus docs are missing the prerequisites section currently. Are you open to advising on these details?

Reply:
@cwarnermm I am not following. What is missing on the Omnibus install page?